.github/workflows/assemble.yml

```diff
@@ -10,27 +10,27 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Configure MapBox Secret Token for Download
+      - name: Configure Mapbox SDK download token
         run: |
           echo "machine api.mapbox.com" >> ~/.netrc
           echo "login mapbox" >> ~/.netrc
           echo "password $MAPBOX_DOWNLOADS_TOKEN" >> ~/.netrc
         env:
           MAPBOX_DOWNLOADS_TOKEN: ${{ secrets.MAPBOX_DOWNLOADS_TOKEN }}
           
-      - name: Configure Ably and MapBox Runtime Secrets for Xcode Build
+      - name: Configure runtime secrets
         run: |
-          echo "MAPBOX_ACCESS_TOKEN = $MAPBOX_ACCESS_TOKEN" >> Secrets.xcconfig
-          echo "ABLY_API_KEY = $ABLY_API_KEY" >> Secrets.xcconfig
-          echo "$AMPLIFY_CONFIGURATION" >> PublisherExample/amplifyconfiguration.json
+          echo "MAPBOX_ACCESS_TOKEN = $MAPBOX_ACCESS_TOKEN" >> Examples/Secrets.xcconfig
+          echo "ABLY_API_KEY = $ABLY_API_KEY" >> Examples/Secrets.xcconfig
+          echo "$AMPLIFY_CONFIGURATION" >> Examples/PublisherExample/amplifyconfiguration.json
         env:
           MAPBOX_ACCESS_TOKEN: ${{ secrets.MAPBOX_ACCESS_TOKEN }}
           ABLY_API_KEY: ${{ secrets.ABLY_API_KEY }}
           AMPLIFY_CONFIGURATION: ${{ secrets.AMPLIFY_CONFIGURATION }}
 
       - name: Build All Projects
         run: |
-          bundle install
-          bundle exec fastlane build_subscriber
-          bundle exec fastlane build_publisher
-          bundle exec fastlane build_example_apps
```

Comment:
what's the reason for removing the bundler?

Reply:
Mainly because there's no need for it, it works fine without. I guess the question, what is the point of bundler. (Fastlane says speed improvement, but doesn't any concrete reasons).

I wanted to propose removing fastlane as well, and just using Makefiles and xcodebuild, as we discussed on Friday. I didn't want to make any more drastic changes. 

LMK if you think I should re-add it. I think it would be nice to remove fastlane (I felt like it was a popular idea in the meeting we had on Friday), perhaps even in this PR then?

Reply:
Using bundler is safer in case of a fastlane version,  because it's using Gemfile.resolved ... but for future use, where we want to remove `fastlane` it's a good idea to remove using bundler for fastlane. LGTM